db/version_set.cc

```diff
@@ -4041,10 +4041,11 @@ Status VersionSet::ExtractInfoFromVersionEdit(
   return Status::OK();
 }
 
-Status VersionSet::GetCurrentManifestPath(std::string* manifest_path) {
+Status VersionSet::GetCurrentManifestPath(std::string* manifest_path,
+  uint64_t* manifest_file_number, const std::string& dbname, Env* env) {
```

Comment:
Nit: assert `manifest_file_number != nullptr` as well. Same for `env`.

Reply:
Added the asserts.